### ISSUE TITLE
Effected Bitcoin stealer

### ISSUE THREAD

**Issue by whomwah** [mysticatea/npm-run-all#154]
Hi, I think this lib is effected by this bitcoin stealer. `flatmap-stream` has now been removed from NPM:

```
└─┬ npm-run-all@4.1.3
  └─┬ ps-tree@1.1.0
    └─┬ event-stream@3.3.6
      └── flatmap-stream@0.1.2
```
https://github.com/dominictarr/event-stream/issues/116
https://www.theregister.co.uk/2018/11/26/npm_repo_bitcoin_stealer/

**Comment by mysticatea** [comment]
Thank you for the report.

However, this is a duplicate of #149, #150, #152, and #153.
I have already published [v4.1.5](https://github.com/mysticatea/npm-run-all/releases/tag/v4.1.5) to address it in the last week.

**Comment by whomwah** [comment]
Sorry 👍
